packages/base-map/src/index.tsx

```diff
@@ -38,13 +38,11 @@ type Props = React.ComponentPropsWithoutRef<React.ElementType> & {
   // Unknown is used here because of a maplibre/mapbox issue with the true type, MapLayerMouseEvent
   onContextMenu?: (e: unknown) => void;
   /** A callback method which is fired when the map zoom or map bounds change */
-  // TODO: does this cause integration issues?
-  onViewportChanged?: (e: maplibregl.MapLibreEvent) => void;
+  onViewportChanged?: (e: State) => void;
```

Comment:
Why was this change made?

Reply:
The params we pass when calling our `onViewportChanged` event were effectively of type `State`.

Reply:
Sounds good